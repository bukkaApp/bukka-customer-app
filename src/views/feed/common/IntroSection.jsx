import React from 'react';

import PropTypes from 'prop-types';
import Navbar from 'src/components/navbar';

import './introsection.scss';

const IntroSection = ({ push }) => (
  <div className="intro-navbar-area">
    <Navbar push={push} bukka />
  </div>
);

export default IntroSection;

IntroSection.propTypes = {
  push: PropTypes.func.isRequired,
};
