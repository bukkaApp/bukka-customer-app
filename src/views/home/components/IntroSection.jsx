import React from 'react';

import PropTypes from 'prop-types';

import PrimaryNavbar from '../../../components/navbar';
import SearchLocation from '../../../components/places-suggest/SearchLocation';

import './introsection.scss';

const LargeTextSection = () => (
  <div className="container large-text-section">
    <h2 className="text-center larger-text">
      Anything, anytime, anywhere. We get it.
    </h2>
  </div>
);

const SmallTextSection = () => (
  <div className="container small-text-section">
    <h5 className="text-center sub-text">
      Food, mart and groceries available for delivery or pickup.
    </h5>
  </div>
);

const SearchLocationSection = () => (
  <div className="container search-location-section">
    <SearchLocation />
  </div>
);

const IntroSection = ({ push }) => (
  <div className="intro-section">
    <PrimaryNavbar push={push} authButton />
    <LargeTextSection />
    <SmallTextSection />
    <div className="col-12 px-0 col-sm-9 col-md-10 col-lg-5 max-search-location mx-auto">
      <SearchLocationSection push={push} />
    </div>
  </div>
);

export default IntroSection;

IntroSection.propTypes = {
  push: PropTypes.func.isRequired,
};

SearchLocationSection.propTypes = {};
