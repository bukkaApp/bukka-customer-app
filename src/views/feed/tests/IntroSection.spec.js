import React from 'react';


import IntroSection from '../common/IntroSection';

describe('IntroSection FeedPage component', () => {
  const wrapper = shallow(<IntroSection push={jest.fn()} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
