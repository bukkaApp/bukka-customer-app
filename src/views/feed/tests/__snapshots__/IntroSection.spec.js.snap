// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroSection FeedPage component renders properly 1`] = `ShallowWrapper {}`;
